### DISCUSSION TITLE
Should we split out the meta-repo from the website repo?

### DISCUSSION THREAD

**Discussion by jacobbieker** [openclimatefix/quartz-frontend#177] [General]
Nowcasting's README details it being the meta repo for all the nowcasting projects. But it also is now where our api/website work is going on as well. I think it would be cleaner if we separated them, but open to what others think.

**Comment by JackKelly** [comment]
I agree 100% that we should split them.

I think this issue might be a duplicate of #92? (Although I'm not certain, so I won't close this issue now!)

**Comment by peterdudfield** [comment]
This has now been done
